src/lib/output/renderer.ts

```diff
@@ -201,7 +201,24 @@ export class Renderer extends ChildableComponent<Application, RendererComponent>
                 }
             }
 
-            this.theme = this.addComponent('theme', new DefaultTheme(this, path));
+            let filename = Path.join(path, 'theme.js');
+            if (!FS.existsSync(filename)) {
+                this.theme = this.addComponent('theme', new DefaultTheme(this, path));
+            } else {
+                try {
+                    const plugin = require(filename);
+                    if(typeof plugin === 'function') {
+                        plugin(this.application);
```

Comment:
For what it's worth, I think there have been very few people that put the theme.js to use and then it wasn't working for a long while anyways.  If we're going to add it back in, then it might as well be consistent with the only current paradigm which is typedoc plugins.  This would mean:

```
   const ThemeClass = require(filename)(this.application);
   this.theme = new (ThemeClass)(path);
```

How does that sound?  For those that have existing theme.js "original flavor," it would be a trivial update.